### ISSUE TITLE
Regular expressions do not function as expected for Chinese text

### ISSUE THREAD

**Issue by Patrick-LuoYu** [languagetool-org/languagetool#6281]
Regular expressions do not function as expected for Chinese text.

For example, assume that the following rule is used:

```xml
<rule id="TEST" name="TEST">
	<regexp>测试</regexp>
	<message>测试</message>
	<example>测试</example>
</rule>
```

The preceding rule should match only "测试". However, "测/试" is also matched, as shown in the following figure.

<img width="587" alt="image" src="https://user-images.githubusercontent.com/76198553/151097734-65d7d9da-1494-49e8-bbd8-651e29fcdbf7.png">

I only tested this problem on [LanguageTool embedded HTTP Server](https://dev.languagetool.org/http-server), where LanguageTool 5.5 was used.

**Comment by Juren39** [comment]
Hello!
I found your problem only arises when you encounter the"/"sign. I think this may be because of the asAnalyzedToken has a problem in chineseTagger method.
![VQSU2XT@Z)8_$34~ {1L A](https://user-images.githubusercontent.com/85175188/164651260-d30bbe5e-1076-49b7-bf64-f9c736ac5285.png)
(Original code)
When we split the "/", we dropped the original "/". So I think we should add a check here to check if the original content of the text was a "/".
After I add a check for "/" , the problem goes away.
![V)PYXH`RF7ZZNUTES VLR4B](https://user-images.githubusercontent.com/85175188/164651926-75207cc9-18ad-4aca-8b02-1805ca4c8e3b.png)
![S(8K@$TLVV2T14B_ WIK@NM](https://user-images.githubusercontent.com/85175188/164651938-5fbc25b6-5261-4f79-83b4-d315c1b8445c.png)

**Comment by SteVio89** [comment]
Fixed by #6572
